src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditorContainer.js

```diff
@@ -107,12 +88,12 @@ class OrderTemplatesEditorContainer extends Component {
     const suffixesSetting = getSettingsList(get(resources, 'suffixesSetting.records', {}));
     const addresses = getAddressOptions(getAddresses(get(resources, 'addresses.records', [])));
     const materialTypes = getMaterialTypesForSelect(resources);
-    const orderTemplate = getOrderTemplatesList(get(resources, 'orderTemplate.records', []));
+    const orderTemplate = get(resources, ['orderTemplate', 'records', 0], {});
     const id = get(match, ['params', 'id']);
     const template = id
-      ? get(orderTemplate, 0, {})
+      ? { orderTemplate }
       : { orderTemplate: INITIAL_VALUES };
-    const title = get(template, 'title') || <FormattedMessage id="ui-orders.settings.orderTemplates.editor.titleCreate" />;
+    const title = get(template, ['orderTemplate', 'templateName']) || <FormattedMessage id="ui-orders.settings.orderTemplates.editor.titleCreate" />;
```

Comment:
are you sure about `'orderTemplate'` here?

Reply:
changed